crates/ra_hir_ty/src/display.rs

```diff
@@ -359,6 +359,21 @@ impl HirDisplay for ApplicationTy {
                     write!(f, ">")?;
                 }
             }
+            TypeCtor::OpaqueType(opaque_ty_id) => {
+                let bounds = match opaque_ty_id {
+                    crate::OpaqueTyId::ReturnTypeImplTrait(func, idx) => {
```

Comment:
Just a style nit (not even a nit, just a personal preference), but for intra-crate links I generally prefer to import things. We need to fix "replace path with import" to also fix usages :-)